docs/3.0rc/deploy/infrastructure-concepts/deploy-via-python.mdx

```diff
@@ -0,0 +1,385 @@
+store your flow code in a remote location, such as a Git repository or cloud storage service.
+
+Using remote storage for your code has several advantages:
+
+1. Faster iterations: You can update your flow code without rebuilding Docker images.
+2. Reduced storage requirements: You don't need to store large Docker images for each code version.
+3. Flexibility: You can use different storage backends based on your needs and infrastructure.
+
+To deploy using remote storage, you'll need to specify where your code is stored by using `flow.from_source` to first load your flow code from a remote location.
+
+Here's an example of loading a flow from a Git repository and deploying it:
+
+```python git-deploy.py
+from prefect import flow
+
+if __name__ == "__main__":
+    flow.from_source(
+        source="https://github.com/username/repository.git",
+        entrypoint="path/to/your/flow.py:your_flow_function"
+    ).deploy(
+        name="my-deployment",
+        work_pool_name="my-work-pool",
+    )
+```
+
+The `source` parameter can be a variety of remote storage options including:
+
+- Git repositories
+- S3 buckets
+- Google Cloud Storage buckets
+- Azure Blob Storage
+
+The `entrypoint` parameter is the path to the flow function within your repository combined with the name of the flow function.
+
+Learn more about remote code storage [here](/3.0rc/deploy/infrastructure-concepts/store-flow-code).
+
+## Set default parameters
+
+You can set default parameters for a deployment using the `parameters` keyword argument in `flow.deploy`.
+
+```python default-parameters.py
+from prefect import flow
+
+@flow
+def my_flow(name: str = "world"):
+    print(f"Hello, {name}!")
+
+if __name__ == "__main__":
+    my_flow.deploy(
+        name="my-deployment",
+        work_pool_name="my-work-pool",
+        # Will print "Hello, Marvin!" by default instead of "Hello, world!"
+        parameters={"name": "Marvin"},
+        image="my-docker-image:dev",
+        push=False,
+    )
+```
+
+Note these parameters can still be overridden on a per-deployment basis.
+
+## Set job variables
+
+You can set default job variables for a deployment using the `job_variables` keyword argument in `flow.deploy`. 
+The job variables provided will override the values set on the work pool.
+
+```python job-variables.py
+import os
+from prefect import flow
+
+@flow
+def my_flow():
+    name = os.getenv("NAME", "world")
+    print(f"Hello, {name}!")
+
+if __name__ == "__main__":
+    my_flow.deploy(
+        name="my-deployment",
+        work_pool_name="my-work-pool",
+        # Will print "Hello, Marvin!" by default instead of "Hello, world!"
+        job_variables={"env": {"NAME": "Marvin"}},
+        image="my-docker-image:dev",
+        push=False,
+    )
+```
+
+Job variables can be used to customize environment variables, resources limits, and other infrastructure options
```

Comment:
```suggestion
[Job variables](/3.0rc/deploy/infrastructure-concepts/customize#override-job-variables) can be used to customize environment variables, resources limits, and other infrastructure options
```

Reply:
There's a link to that page at the end of this section. I like it at the end so that the flow is you read through this and then go read more if you want/need to.